migrations/witnet.singletons.js

```diff
@@ -1,17 +1,17 @@
 module.exports = {
   WitnetBytecodes: {
-    salt: 69620662,
+    salt: 178366606,
   },
-  WitnetPriceRouter: {
-    salt: 35082159,
+  WitnetPriceFeeds: {
+    salt: 7148124,
   },
   WitnetRandomness: {
-    salt: 103419408,
+    salt: 36274595,
   },
   WitnetRequestBoard: {
-    salt: 72167531,
+    salt: 21096971,
   },
   WitnetRequestFactory: {
-    salt: 84037617,
-  }
+    salt: 222309,
+  },
```

Comment:
Where did you source this salts from?

Reply:
E.g.:
`$ truffle exec scripts/vanitygen.js --network polygon.zkevm.goerli --artifact WitnetRequestRandomness --target 0xfabada`